dist/mjs/interface/interface.d.ts

```diff
@@ -12,8 +19,8 @@ export interface NodeStatistics {
         total_tasks: number;
         timed_out_count: number;
         last_timed_out: string;
-        last_successful_proof_task_time: string;
-        last_successful_proof_processing_time: number;
+        setup_timing_stats?: TimingStatistics;
```

Comment:
This is not backward compatible as old db has the data last_successful_proof_task_time and last_successful_proof_processing_time and will not be able deserialize?